code/datums/wounds/cuts.dm

```diff
@@ -141,6 +141,18 @@
 		to_chat(user, "<span class='warning'>You're already interacting with [victim]!</span>")
 		return
 
+	if(user.is_mouth_covered())
+		to_chat(user, "<span class='warning'>Your mouth is covered, you can't lick [victim]'s wounds!</span>")
+		return
+
+	if(!user.getorganslot(ORGAN_SLOT_TONGUE))
+		to_chat(user, "<span class='warning'>You can't lick wounds without a tongue!</span>") // f in chat
+		return
+
+	// transmission is one way patient -> felinid since google said cat saliva is antiseptic or whatever, and also because felinids are already risking getting beaten for this even without people suspecting they're spreading a deathvirus
+	for(var/datum/disease/D in victim.diseases)
+		user.ForceContractDisease(D)
```

Comment:
I forget, would the ForceContractDisease() proc force the felinid's virus to replace the lick-ee's virus, even if the former's transmission stat was lower than the latter's resistance stat?

Reply:
No